docs/framework/resources/creating-satellite-assemblies-for-desktop-apps.md

```diff
@@ -89,8 +89,13 @@ The following is a simple "Hello world" example that displays a message box cont
 
 2. To indicate that English (en) is the application's default culture, add the following <xref:System.Resources.NeutralResourcesLanguageAttribute?displayProperty=nameWithType> attribute to the application's AssemblyInfo file or to the main source code file that will be compiled into the application's main assembly.
 
-    [!code-csharp[Conceptual.Resources.Locating#2](~/samples/snippets/csharp/VS_Snippets_CLR/conceptual.resources.locating/cs/assemblyinfo.cs#2)]
-    [!code-vb[Conceptual.Resources.Locating#2](~/samples/snippets/visualbasic/VS_Snippets_CLR/conceptual.resources.locating/vb/assemblyinfo.vb#2)]  
+    ```csharp
+    [assembly: NeutralResourcesLanguageAttribute("en")]
+    ```
+
+    ```vb
```

Comment:
Perhaps we should also remove the `<snippet2>` tags from the .vb file if that 's the last reference to it.

Reply:
Yes that's the last reference. I'll update it.